tasks/main.yml

```diff
@@ -85,10 +85,16 @@
       validate:
         fail_on_error: yes
         strict: yes
-    register: deploy_account
+
+  - name: Get deploy account name
+    k8s:
+      kind: ServiceAccount
+      namespace: "{{ k8s_namespace }}"
+      name: "deploy-account"
```

Comment:
I think this should include `host: "{{ k8s_auth_host }}"` like the other tasks above this to make sure we use the project K8s API host